peekingduck/configs/model/posenet.yml

```diff
@@ -51,3 +51,4 @@ model_type: resnet # 50, 75, 100 or resnet
 resolution: { height: 225, width: 225 }
 max_pose_detection: 10
 score_threshold: 0.4
+use_jit: true
```

Comment:
Similarly, to default to `False` here too.

Reply:
Set default to false and added documentation for `use_jit` in the node's docstring